<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="spectrum_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="Top">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bottom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>bottom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Red">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Orange">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>60</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Orange</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Yellow">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>60</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Yellow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Green">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>60</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Blue">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>60</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Navy">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Navy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Violet">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>60</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Violet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="White">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>100</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Black">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Gray">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>100</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Gray</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Yellow_2">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>250</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Yellow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="White_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Green_2">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>250</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Violet_2">
   <property name="geometry">
    <rect>
     <x>468</x>
     <y>250</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Violet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Orange_2">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>250</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Orange</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Black_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Gray_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Gray</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Red_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Blue_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Navy_2">
   <property name="geometry">
    <rect>
     <x>389</x>
     <y>250</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Navy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spectrum_button</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spectrum_button</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
